CHANGELOG.md

```diff
@@ -62,6 +62,7 @@
 - Added support for `Series.dt.normalize` and `DatetimeIndex.normalize`.
 - Added support for `Index.is_boolean`, `Index.is_integer`, `Index.is_floating`, `Index.is_numeric`, and `Index.is_object`.
 - Added support for `DatetimeIndex.round`, `DatetimeIndex.floor` and `DatetimeIndex.ceil`.
+- Added support for lazy `TimedeltaIndex`.
```

Comment:
I think it makes more sense to add this a sub-point under the "Added limited support for `Timedelta` type including" a couple of lines above.